temporalio/worker/_workflow.py

```diff
@@ -193,6 +206,43 @@ async def _handle_activation(
     ) -> None:
         global LOG_PROTOS
 
+        def get_traceback_for_thread(
+            thread_id: Optional[int],
+        ) -> Tuple[Optional[TracebackType], bool]:
```

Comment:
Should have a docstring explaining what the components of the return type are (the bool isn't immediately clear for example)